### PR TITLE
Add arrow nav to maps menu + fix server browser nav

### PR DESCRIPTION
**Pull request by F1F7Y**
- Selects 1st server when you open server browser
  - Allows for full keyboard / controller navigation
- Adds arrow nav to maps menu
- Moves Player / Server counts

### CODE REVIEW

**Review by abarichello — CHANGES_REQUESTED** [review]
This new map menu is looking really good, please fix the formatting issues so we can use it in 1.6.0

File: Northstar.Client/mod/resource/ui/menus/panels/map_grid.res [hunk 1]
```diff
@@ -1,201 +1,460 @@
...
     tall 					165
 		xpos					2
 		
-		Classname			MapGridButtons
+		Classname			MapGridInfo
 		scriptID			11
 		
 		controlSettingsFile	"resource/ui/menus/panels/map_grid_button.res"
 		
+		pin_to_sibling					GridInfo1x3
```
[the remark this answers is not in the document]
Comment (F1F7Y) [reply Northstar.Client/mod/resource/ui/menus/panels/map_grid.res:197]: :dread:

File: Northstar.Client/mod/scripts/vscripts/ui/menu_map_select.nut [hunk 2]
```diff
@@ -137,9 +184,11 @@ void function MapButton_Activate( var button )
 
 void function MapButton_Focus( var button )
 {
-	int mapID = int( Hud_GetScriptID( Hud_GetParent( button ) ) )
```
[the remark this answers is not in the document]
Comment (F1F7Y) [reply Northstar.Client/mod/scripts/vscripts/ui/menu_map_select.nut:140]: wat

File: Northstar.Client/mod/scripts/vscripts/ui/menu_map_select.nut [hunk 3]
```diff
@@ -94,11 +102,50 @@ void function OnOpenMapsMenu()
...
+	UpdateMapsGrid()
+	UpdateListSliderPosition()
+	UpdateNextMapInfo()
+	
+	Hud_SetFocused( file.gridButtons[ file.lastSelectedID ] )
+}
+
+void function OnHitDummyBottom( var button )
+{
+	if ( file.mapsArrayFiltered.len() <= BUTTONS_PER_PAGE || file.mapsArrayFiltered.len() <= 12 ) return
```
[the remark this answers is not in the document]
Comment (F1F7Y) [reply Northstar.Client/mod/scripts/vscripts/ui/menu_map_select.nut]: should be fine now

**Review by abarichello — APPROVED** [review]
Thanks

**Pull request merged**